src/wallet/mod.rs

```diff
@@ -3692,7 +3714,7 @@ pub(crate) mod test {
             .unwrap();
 
         let mut builder = wallet.build_fee_bump(txid).unwrap();
-        builder.fee_rate(FeeRate::from_sat_per_vb(141.0));
+        builder.fee_rate(FeeRate::from_sat_per_vb(142.0));
```

Comment:
Ok so my understanding is the feerate is being changed here to keep round test values. Is there some analytic way we could calculate `142.0` here so it doesn't look like "just set it to whatever to make the test pass".

Reply:
Just for context: the value of the feerate here must be high enough that during rbf we are forced to add a new input and also that we have to remove the change that we had previously.

So in theory:

```
new_tx_weight = tx.weight() + extra_input_weight + input_satisfaction_weight - change_output_weight
fee_abs = 750000 - 45000 - epsilon (10, 20 sats?)
fee_rate = fee_abs / new_tx_weight
```

Note that the `tx.weight()` should be measured *after* we push the fake signatures, or alternatively an additional `input_satisfaction_weight` should be added to the `new_tx_weight` to account for that.